lib/pipeline.js

```diff
@@ -23,13 +29,14 @@ class Pipeline extends Readable {
     })
   }
 
-  clone ({ basePath, context, objectMode, variables }) {
+  clone ({ basePath, context, objectMode, variables, logSink }) {
     return new Pipeline(this.node, {
       basePath: basePath || this.basePath,
       context: context || this.context,
       variables: variables || this.variables,
       loaderRegistry: this.loaderRegistry,
-      objectMode: objectMode || this.objectMode
+      objectMode: objectMode || this.objectMode,
+      logSink
```

Comment:
Is there a specific reason to call it `logSink` and not just `log` like in the rest of the code?

Reply:
Not at all. Will rename